programs/cleos/httpc.hpp

```diff
@@ -107,7 +107,9 @@ namespace eosio { namespace client { namespace http {
 
 
    const string history_func_base = "/v1/history";
+   const string trace_api_func_base = "/v1/trace_api";
    const string get_actions_func = history_func_base + "/get_actions";
+   const string get_transaction_trace_func = trace_api_func_base + "/get_transaction_trace";
```

Comment:
I think we should add `trace_api_func_base + "/get_block` since we are adding this `/v1/trace_api` option. Call it `cleos get block_trace`